doc/source/cluster/running-applications/monitoring-and-observability.rst

```diff
@@ -188,3 +188,14 @@ Ray Cluster information. Here, we will use a Python script and the
                     'object_store_memory': 2.0},
      'alive': True}]
     """
+
+Setting up Grafana and Metrics
+##############################
+
+One common pattern of supporting :ref:`metrics <ray-metrics>` on a remote ray cluster is by running the Prometheus and Grafana services
+on the head node of the cluster. However, in order to view the :ref:`Dashboard <ray-dashboard>` metrics on your local
+machine, you must configure the Dashboard UI to embed the metrics graphs via a public address for the Grafana instance.
+
+The `RAY_GRAFANA_HOST` env var can be set when launching Ray to configure how the Dashboard UI embeds the metrics.
```

Comment:
Can you add this example to `ray-metrics` page and just have a link here instead?